test/common/formatter/substitution_format_string_test.cc

```diff
@@ -86,10 +86,11 @@ TEST_F(SubstitutionFormatStringUtilsTest, TestInvalidConfigs) {
   };
   for (const auto& yaml : invalid_configs) {
     TestUtility::loadFromYaml(yaml, config_);
-    EXPECT_THROW_WITH_MESSAGE(SubstitutionFormatStringUtils::fromProtoConfig(config_, context_),
-                              EnvoyException,
-                              "Only string values, nested structs, list values and number values "
-                              "are supported in structured access log format.");
+    EXPECT_THROW_WITH_MESSAGE(
```

Comment:
Little weird to have done this before de-exceptioning `SubstitutionFormatter` which could have cascaded up into the return value from `fromProtoConfig`, but okay I guess.